Update index.md

juliasourceress opened the pull request.

dadlerj reviewed (approved):
Thanks for updating. But no need for the special description and honorifics there 😄 . I'm fine with or without, or with something like "(temporary)" on my name instead. Totally up to you.

dadlerj commented on handbook/ce/index.md:
```diff
@@ -20,9 +20,10 @@ TODO
-- [Christine Lovett](../../company/team/index.md#christine-lovett-she-her)
-- [Tion Thomas](../../company/team/index.md#tion-thomas-he-him)
+- [Julia Gilinets](../../company/team/index.md#julia-gilinets-sheher) (VP of CE)
+- [Dan Adler](../../company/team/index.md#dan-adler-hehim) (Sourcegraph's first ever CE and previous CE team manager)
```
```suggestion
- [Dan Adler](../../company/team/index.md#dan-adler-he-him) (interim CE member, former CE manager)
```

dadlerj commented:
Just noticed this wasn't merged. Fixed merge conflicts and updated my title. LMK if you're okay with this @juliasourceress !

juliasourceress commented:
👍🏻
Thank you! 

> On Sep 23, 2020, at 2:44 PM, Dan Adler <notifications@github.com> wrote:
> 
> ﻿
> Just noticed this wasn't merged. Fixed merge conflicts and updated my title. LMK if you're okay with this @juliasourceress !
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.
